styles/variables-silicon.less

```diff
@@ -484,7 +484,7 @@
 
 // WindDirectionControl
 // ---------------------------------------
-@agate-winddirectioncontrol-icon-font-size: 210px;
-@agate-winddirectioncontrol-icon-height: 210px;
-@agate-winddirectioncontrol-icon-line-height: 210px;
-@agate-winddirectioncontrol-icon-width: 210px;
+@agate-winddirectioncontrol-icon-font-size: 240px;
+@agate-winddirectioncontrol-icon-height: 240px;
+@agate-winddirectioncontrol-icon-line-height: 240px;
+@agate-winddirectioncontrol-icon-width: 240px;
```

Comment:
I think the size of the icon and line-height are 120 px and the margin should be 0.
![image](https://user-images.githubusercontent.com/5037429/115181515-17c26b00-a113-11eb-85a8-7a41e0dfa2c7.png)

Reply:
@MikyungKim 
If I use a font-size of 120px and a line height of 120px it looks like this:
![image](https://user-images.githubusercontent.com/77739822/115353524-4860de00-a1c1-11eb-8baf-7e2a2fa805d2.png)
The icon is very small and because of the line height it is no longer centered.

With the font-size of 240px and line height of 240px it looks like this:
![image](https://user-images.githubusercontent.com/77739822/115353793-9aa1ff00-a1c1-11eb-829a-b917ea6aa078.png)

And this is the comparison of the 240px size and the GUI
![image](https://user-images.githubusercontent.com/77739822/115354485-58c58880-a1c2-11eb-9e7d-002cdf3c41d7.png)

Reply:
I didn't mean to change `font-size`. I meant the size and line-height to be 120px. Please re-check.

Reply:
That's why I dragged those three lines(488 to 490) to comment on this.

Reply:
I solved the issues and removed the margin, but I could only remove he margin by adding it inside of 'applySkins' otherwise it would not apply.